rust/cloud-storage/Dockerfile.convert_service.prebuilt

```
@@ -0,0 +1,39 @@
+FROM ubuntu:noble AS base
+ARG SERVICE_NAME=convert_service
+WORKDIR /app
+
+# Accept the Microsoft core fonts license
+RUN echo "ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true" | debconf-set-selections
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)